<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addClientButton">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>110</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>addClientButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateClientButton">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>160</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>updateClientButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteClientButton">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>220</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>deleteClientButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="prenomLineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="telephoneLineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLineEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>480</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
